### ISSUE TITLE
operator does not upgrade packagserver

### ISSUE THREAD

**Issue by sjenning** [operator-framework/operator-lifecycle-manager#706]
After a cluster upgrade from `4.0-2019-02-08-055616` to `4.0.0-0.alpha-2019-02-08-131655`, the `packageserver` deployment is not upgraded
```
$ oc get clusterversion
NAME      VERSION                           AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.0.0-0.alpha-2019-02-08-131655   True        True          11m

$ oc get deployment olm-operator -oyaml
...
spec:
...
  template:
...
    spec:
      containers:
      - args:
        - -writeStatusName
        - operator-lifecycle-manager
        command:
        - /bin/olm
        env:
        - name: OPERATOR_NAMESPACE
          valueFrom:
            fieldRef:
              apiVersion: v1
              fieldPath: metadata.namespace
        - name: OPERATOR_NAME
          value: olm-operator
        image: registry.svc.ci.openshift.org/openshift/origin-v4.0-2019-02-08-131655@sha256:c379e068eca7768313ed026f8fac04057920526d388c42dfd585a3a0c4227677

$ oc get deployment packageserver -oyaml
...
spec:
...
  template:
...
    spec:
      containers:
      - command:
        - /bin/package-server
...
        image: registry.svc.ci.openshift.org/openshift/origin-v4.0-2019-02-08-055616@sha256:c379e068eca7768313ed026f8fac04057920526d388c42dfd585a3a0c4227677
```
@smarterclayton @derekwaynecarr

**Comment by njhale** [comment]
@sjenning The packageserver is packaged as an OLM operator. This means it's defined as a CSV in a configmap backed catalog and deployed via OLM Subscription. These are all defined in our top level `./manifests` directory and should be getting replaced by CVO. It would help a lot if you could post the on cluster `olm-operators` ConfigMap as well as the backing ConfigMap of the same name.

**Comment by jpeeler** [comment]
I was able to reproduce this and saw that the olm-operators configmap is getting updated, but the CSV/deployment is not.

**Comment by njhale** [comment]
This most likely has to do with us not generating a "replacement" CSV for packageserver - OLM doesn't know to upgrade the CSV.

**Comment by ecordell** [comment]
Fixed by #722
